lib/factory_girl/linter.rb

```diff
@@ -1,11 +1,12 @@
 module FactoryGirl
   class Linter
-    def self.lint!(factories_to_lint)
-      new(factories_to_lint).lint!
+    def self.lint!(factories_to_lint, options)
```

Comment:
instead of having the conditionals spread across this class, thoughts on having there be two linters, one for factories and one for traits, or having two class methods and having the things being linted have a more standard interface?